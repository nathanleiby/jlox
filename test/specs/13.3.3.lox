class Eclair {
  cook() {
    super.cook();
    // NOTE: I removed accented chars
    print "Pipe full of creme patissiere.";
  }
}
