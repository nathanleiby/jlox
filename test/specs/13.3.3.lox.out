Error: Cannot use 'super' in a class with no superclass.
